src/test/jtx/impl/Env.cpp

```diff
@@ -136,9 +136,15 @@ Env::close(
         auto resp = rpc("ledger_accept");
         if (resp["result"]["status"] != std::string("success"))
         {
-            JLOG(journal.error())
-                << "Env::close() failed: " << resp["result"]["status"]
-                << std::endl;
+            std::string reason = "internal error";
+            if (resp.isMember("error_what"))
+                reason = resp["error_what"].asString();
+            else if (resp.isMember("error_message"))
+                reason = resp["error_message"].asString();
+            else if (resp.isMember("error"))
+                reason = resp["error"].asString();
```

Comment:
Good question.  For all of the current cases the `"error_what"` gives the best answer.  I included the other two cases as backup, but it would be entirely reasonable to tell me to take them out since there is no evidence they are needed.

I removed the `["result"]["status"]` case because in every case I had it returned `NULL`.  Apparently it used to be useful, but it's currently not useful.  That said, it's hard to predict the future...

Reply:
I don't think it hurts to leave the backups, and if you haven't seen any cases where `status` is useful, then there's no need to add it. If that changes in the future, we know where to change.

My hunch is that whoever wrote the original error log knew that they had to check `status` for `success`, but didn't consider or test failure cases, so they never really saw it emit the unhelpful message. That's not _great_, but it's test code, so it's not the end of the world either.